generative_ai/embedding_batch.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud.aiplatform import BatchPredictionJob
+
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+OUTPUT_URI = os.getenv("GCS_OUTPUT_URI")
+
+
+def embed_text_batch() -> BatchPredictionJob:
+    """Example of how to generate embeddings from text using batch processing.
+
+    Read more: https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/batch-prediction-genai-embeddings
+    """
+    # [START generativeaionvertexai_sdk_embedding_batch]
+    import vertexai
+    from vertexai.preview.language_models import TextEmbeddingModel
```

Comment:
import `language_models` as the module rather than `TextEmbeddingModel` class directly.

https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
That convention hasn't been followed for most of the Generative AI Samples. I think it makes sense to be consistent with the existing samples

Reply:
This is a Google-wide standard so I'd strongly push back on this against keeping it consistent. There are no advantages to importing classes only.